source/simulator/assembly.cc

```diff
@@ -984,6 +1012,14 @@ namespace aspect
 
 
 
+  template <int dim>
+  Simulator<dim>::Assemblers::Properties::Properties ()
+    :
+    need_face_material_model_data (false)
```

Comment:
Ah, maybe because `needed_update_flags` is left entirely uninitialized. Will update the patch by an initialization step...